db/schema.rb

```diff
@@ -22,6 +22,7 @@
     t.string   "avatar"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string   "login"
```

Comment:
what does this represent? from reading this, it's not easy to tell what is stored in this column